src/controller/task.ts

```diff
@@ -12,6 +13,10 @@ export default {
         importance,
         status
       })
+      Logger.info('Task created', {
+        id: newTask._id,
+        statusCode: res.statusCode
```

Comment:
statusCode does not need to be logged (except errorHandler)

Reply:
fixed e9acee3ca21fc4f0db8449487a6402632f34093e